vignettes/climr_workflow_beg.Rmd

```diff
@@ -40,64 +40,89 @@ cpath <- if (interactive()) {
 options("climr.cache.path" = cpath)
 ```
 
+###Using the `climr_downscale()` function
```

Comment:
please add space after "###"

Reply:
(note that this is simply to follow good practices suggested  by people like H Wickam and friends)